regulations/static/regulations/js/source/helpers.js

```diff
@@ -38,6 +38,17 @@ if (!Array.prototype.indexOf) {
   };
 }
 
+$(window).load(function () {
+
+  // notice and comment homepage expandable section
+  $('.nc-homepage .expandable .button').click(function (){
```

Comment:
Does this have to be tied to the notice-and-comment home page? The logic seems pretty general purpose?

Related: this is very similar to the `show-more-toggle` logic. Would it make sense to have a more general purpose function that does both?